bioblend/toolshed/repositories/__init__.py

```diff
@@ -35,6 +35,72 @@ def get_repositories(self):
         """
         return Client._get(self)
 
+    def search_repositories(self, q, page=1, page_size=10):
+        """
+        Search for repositories in a Galaxy Tool Shed
+
+        :type  q: str
+        :param q: query string for searching purposes
+
+        :type  page: str
```

Comment:
It is being cast to float anyways so 1 and '1'  should both work. Do you think putting int here makes more sense? If so I will change it.